src/Gui/ConversionUtils.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
```

Comment:
For new files, please add the license banner. You can copy the banner from any other pytocs file.